src/services/auth/plugins/magicLink/magicLink.test.ts

```diff
@@ -234,6 +234,22 @@ describe('Auth routes tests', () => {
       expect(response.statusMessage).toEqual(ReasonPhrases.BAD_REQUEST);
       expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST);
     });
+
+    it('Bad request if the username contains special characters', async () => {
+      const email = 'test@tust.lol';
```

Comment:
```suggestion
      const email = faker.internet.email().toLowerCase();
```

Reply:
Nice one ! Faker internet is my favourite internet.